include/IECorePython/GeometricTypedDataBinding.inl

```diff
@@ -42,6 +42,31 @@
 
 #include <sstream>
 
+namespace
+{
+	std::string interpretationStr( IECore::GeometricData::Interpretation interpretation )
+	{
+		switch( interpretation )
+		{
+			case IECore::GeometricData::Numeric: // None
```

Comment:
`Numeric` is something we're phasing out in favour of `None`, so we should serialise as `"None"` to avoid prolonging its life any longer.

Reply:
fixed in 03ea16f3